components/GrommetMascot.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+import { Box, Image, Text } from "grommet";
+
+export const GrommetMascot = () => (
+    <Box direction="row" pad={{ top: "large" }}>
```

Comment:
`direction="row"` for large containers could behave tricky when trying to apply responsive behavior. Can you confirm it looks as expected?